### PR TITLE
Run instrumentation tests

### LINKED ISSUE #301: Add integration tests to Travis config

### PR DESCRIPTION
**Pull request by marwanad**
Closes #301.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,8 +13,14 @@ android:
 jdk:
   - oraclejdk8
 
-script:
-  - ./gradlew clean build -x checkstyleTest --stacktrace
+#script:
+#  - ./gradlew clean build -x checkstyleTest --stacktrace
+
+before_script:
+  - echo no | android create avd --force -n test -t android-18 --abi armeabi-v7a
```
Comment (digitalbuddha) [inline .travis.yml]: is 18 our min?

Comment (marwanad) [inline .travis.yml]: should be 15.

**Comment by marwanad** [comment]
@digitalbuddha relevant line from logs: https://travis-ci.org/apollographql/apollo-android/builds/209199191#L1799

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +14,13 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean build -x checkstyleTest --stacktrace
+  - ./gradlew build connectedCheck -x checkstyleTest --stacktrace
```
Comment (digitalbuddha) [inline .travis.yml]: is connectedCheck not part of build? I would think build runs everything.

Comment (marwanad) [inline .travis.yml]: That's what I thought too but according to http://tools.android.com/tech-docs/new-build-system/user-guide

> build does not depend on deviceCheck, or connectedCheck.

**Pull request merged**